pkg/whip/server.go

```diff
@@ -151,7 +152,60 @@ func (s *WHIPServer) Start(
 
 	// Trickle, ICE Restart unimplemented for now
 	r.HandleFunc("/{app}/{stream_key}/{resource_id}", func(w http.ResponseWriter, r *http.Request) {
-		w.WriteHeader(http.StatusNotImplemented)
+		w.Header().Set("Access-Control-Allow-Origin", "*")
+
```

Comment:
I added a really simple ETag. We could add another dependency, but just wanted to keep it simple.

We don't need ETag yet. In the future if we want to support clients trickling candidates it will matter. At this point ETag is never used/useful.

Reply:
Thanks. How about the "If-Match" header handling to differentiate between Tricke-ICE and ICE restart? i.e, returning a "422 Unprocessable Content" error if the "If-Match" header is set to anything else than '*'? (see RFC section 4.1.1 and 4.1.3)